runtime/usr/local/lib/python3.4/dist-packages/taupage/__init__.py

```diff
@@ -126,10 +127,17 @@ def get_config(filename=TAUPAGE_CONFIG_PATH):
 
 
 def get_token(config: dict, token_name: str, scopes: list):
+    oauth_access_token_url = config.get('oauth_access_token_url')
     token_url = config.get('token_service_url')
 
+    if not oauth_access_token_url:
+        logging.warning('No OAuth access token URL configured in Taupage YAML ("oauth_access_token_url" property)')
+
     if not token_url:
         logging.warning('No token service URL configured in Taupage YAML ("token_service_url" property)')
+
+    if not oauth_access_token_url and not token_url:
+        # neither of the URLs is given, no chance to continue
```

Comment:
@prayerslayer I think the interface of get_token (as is right now) should return None (return without anything will return None) in case prerequisites might be missing